test/Bridges/Variable/flip_sign.jl

```diff
@@ -33,6 +33,59 @@ config = MOIT.TestConfig()
         @test MOI.get(bridged_mock, MOI.VariablePrimalStart(), y) == 1
     end
 
+    @testset "Test Mock model" begin
+        var_names = ["a", "b", "c", "d"]
+        MOI.set(mock, MOI.VariableName(), MOI.get(mock, MOI.ListOfVariableIndices()), var_names)
```

Comment:
Only set the name of the bridged variables, the other variables should have the names `x`, `z` and `w` already set.